docs/source/pillars/data_management.md

```diff
@@ -24,6 +24,11 @@ _The ability of the TRE organisation to manage how and where data is stored, how
 * - Statement
   - Guidance
   - Importance
+* - You have a process in place to assess the legal and regulatory implications of handling the data through its full lifecycle together with any required security controls.
```

Comment:
```suggestion
* - You have a process in place to assess the legal and regulatory implications of handling the data through its full lifecycle together with any required security controls.
```

This means security controls required by nature of the data?

Reply:
Rephrased